docs/contributing/contributing-to-airbyte/README.md

```diff
@@ -52,19 +99,36 @@ To see what has already been proposed by the community, you can look [here](http
 
 Watch out for duplicates! If you are creating a new issue, please check [existing open](https://github.com/airbyte.io/airbyte/issues), or [recently closed](https://github.com/airbytehq/airbyte/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20). Having a single voted for issue is far easier for us to prioritize
 
-## **Requesting new integrations**
+### **Requesting new integrations**
 
 This is very similar to requesting new features. The template will change a bit and all integration requests will be tagged with the “**community\_new**” and “**area/integration**” labels.
 
 To see what has already been proposed by the community, you can look [here](https://github.com/airbytehq/airbyte/labels/area%2Fintegration). Again, watch out for duplicates!
 
-## **Reporting bugs**
+### **Reporting bugs**
 
 **‌**Bug reports help us make Airbyte better for everyone. We provide a preconfigured template for bugs to make it very clear what information we need.
 
 ‌Please search within our [already reported bugs](https://github.com/airbytehq/airbyte/issues?q=is%3Aissue+is%3Aopen+label%3Atype%2Fbug) before raising a new one to make sure you're not raising a duplicate.
 
-## **Reporting security issues**
+### **Reporting security issues**
 
 Please do not create a public GitHub issue. If you've found a security issue, please email us directly at [security@airbyte.io](mailto:security@airbyte.io) instead of raising an issue.
 
+## **6. Review process**
+
+**‌‌**If you are considering adding to the codebase or contributing a new integration, first a big thank you. We sincerely appreciate your help in our mission to solve the data integration problem!
```

Comment:
```suggestion
**‌‌**If you are considering adding to the codebase or contributing a new integration: a big thank you! We sincerely appreciate your help.
```

Reply:
suggested phrasing doesn't make use of "our" mission since it could imply it's not also theirs ;)